pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -409,7 +431,7 @@ func (c *Controller) queueServiceForEndpointSlice(endpointSlice *discovery.Endpo
 		utilruntime.HandleError(fmt.Errorf("Couldn't get key for EndpointSlice %+v: %v", endpointSlice, err))
 		return
 	}
-	c.queue.Add(key)
+	c.queue.AddAfter(key, endpointSliceChangeSyncDelay)
```

Comment:
Shouldn't we use: c.endpointUpdatesBatchPeriod instead?

Reply:
that defaults to 0, which means no batching, which probably isn't what we want, right?

Reply:
I'm not sure - both places are kind-of for batching. So don't we want to be consistent? Either we do batching (in different places) or don't do it?

Reply:
Pod updates are the normal case, so being super-responsive by default and allowing opting into longer batch periods makes sense. Reacting to endpointslice updates ensures we recover and correct changes made out-of-band, so batching by default makes more sense to me. I could see taking the max of 1s or c.endpointUpdatesBatchPeriod.

Reply:
OK - makes sense.
And I agree that max of these two values would be even better.

Reply:
👍 that all makes sense, I've updated the code to use the max of those two values.